x/cert/simulation/genesis.go

```diff
@@ -17,5 +17,7 @@ func RandomizedGenState(simState *module.SimulationState) {
 		}
 	}
 
+	gs.NextCertificateId = 1
```

Comment:
I think just setting gs := DefaultGenesisState() should be better

Reply:
Actually there're a bunch of certifiers inserted into genesis above, so I guess `DefaultGenesis()` is not very necessary.